Makefile

```diff
@@ -595,9 +595,9 @@ ifdef CI
 	cp bin/windows_amd64/roxctl.exe image/bin/roxctl-windows-amd64.exe
 else
 ifneq ($(HOST_OS),linux)
-	cp bin/linux_$(GOARCH)/roxctl image/bin/roxctl-linux-$(GOARCH)
+	cp bin/linux_amd64/roxctl image/bin/roxctl-linux-amd64
 endif
-	cp bin/$(HOST_OS)_amd64/roxctl image/bin/roxctl-$(HOST_OS)-amd64
+	cp bin/$(HOST_OS)_$(GOARCH)/roxctl image/bin/roxctl-$(HOST_OS)-$(GOARCH)
 endif
```

Comment:
Okay, I think this entire thing (lines 596-601) should be simply:

```Makefile
else
	cp bin/linux_$(GOARCH)/roxctl image/bin/roxctl-linux-$(GOARCH)
endif
```

Am I wrong?

Reply:
Please check https://github.com/stackrox/stackrox/pull/3051

Reply:
I would agree, yes